Tests/SwiftDocCTests/Test Bundles/SampleBundle.docc/MySample.md

```diff
@@ -0,0 +1,9 @@
+# MySample
+
+@Metadata {
+    @CallToAction(url: "https://example.com/sample.zip", purpose: download)
```

Comment:
Should/do we have a test for the `link` purpose?

Reply:
The `testValidDirective` test in `CallToActionTests.swift` at least ensures that the `link` purposes parses correctly. It looks like i don't check for the expected button label, though, so i could extend it to check for that if you think it would be helpful.

Reply:
Ah I was more concerned about making sure we have a test for ensuring an external link makes it all the way through to the RenderJSON – but actually this test case already covers that doesn't it. I was just confusing the `url` versus `file` case with the `purpose`. So I think we're good – covering the `purpose` couldn't hurt though! Up to you.